components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -366,9 +367,11 @@ - (nullable NSString *)accessibilityLabel {
   NSMutableArray *labelComponents = [NSMutableArray array];
 
   // Use untransformed title as accessibility label to ensure accurate reading.
```

Comment:
Nit: Would you please update the comment here so it more accurately reflects the logic? Something like, "Defaults to the `accessibilityLabel` of the UITabBarItem, if available, else the untransformed title."